homeworks/17/instagram-api/Post.java

```diff
@@ -0,0 +1,173 @@
+package test;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class Post
+ */
+@WebServlet("/posts")
```

Comment:
This will only match the exact URL ending in "/posts", there should have been "/posts/{id}/comments", etc.